code/WorkInProgress/MechanicMadness.dm

```diff
@@ -3139,5 +3139,68 @@
 	name = ""
 	icon = 'icons/misc/mechanicsExpansion.dmi'
 	icon_state = "connectionArrow"
+
+
+/obj/item/mechanics/screen
+	name = "Letter Display Component"
+	desc = ""
+	icon_state = "comp_screen"
+	cabinet_banned = true
+
+	var/letter_index = 1
+	var/display_letter = null
+
+	get_desc()
+		. = ..()
+		. += "<br><span class='notice'>Letter Index: [src.letter_index]"
+		if (src.level == 2 || src.display_letter != null)
+			. += " | Currently Displaying: '[src.display_letter]'"
+		. += "</span>"
+	secure()
+		src.display(" ")
+	loosen()
+		src.display_letter = null
+		src.icon_state = "comp_screen"
+	New()
+		..()
```

Comment:
any reason why not `. = ..()`?

Reply:
It's `New()`, I wouldn't do `. = ..()` there.

Reply:
fair enough

Reply:
Fairly new to the language.  What's `. = ..()` and when would I add it?

Reply:
Sets the default return value of the overridden proc to the return value of the parent proc: useful when things return meaningful values

Reply:
Got it.  Thanks!